core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -420,7 +420,11 @@ public ClusterState execute(ClusterState currentState) throws Exception {
                             .put(indexMetaData, false)
                             .build();
 
-                    logger.info("[{}] creating index, cause [{}], templates {}, shards [{}]/[{}], mappings {}", request.index(), request.cause(), templateNames, indexMetaData.numberOfShards(), indexMetaData.numberOfReplicas(), mappings.keySet());
+                    String maybeShadowIndicator = IndexMetaData.isIndexUsingShadowReplicas(indexMetaData.settings()) ? "s" : "";
+                    logger.info("[{}] creating index, cause [{}], templates {}, shards [{}]/[{}], mappings {}",
+                            request.index(), request.cause(), templateNames, indexMetaData.numberOfShards(),
+                            indexMetaData.numberOfReplicas() + maybeShadowIndicator,
```

Comment:
couldn't you do `{}{}` and `indexMetaData.numberOfReplicas(), maybeShadowIndicator` instead?